### DISCUSSION TITLE
Add popover to TimeInput

### DISCUSSION THREAD

**Discussion by converter-user-old** [mantinedev/mantine#1052] [Feature requests]
### Usage example, including component/hook API

Like this (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time):

![image](https://user-images.githubusercontent.com/85690286/148059138-ddca23ab-f023-45c3-b0a1-0f3f18bb8c9a.png)

**Comment by UltraInstinct05** [comment]
A sliding reel maybe complicated to implement. A dropdown seems easier. As an example https://demos.telerik.com/kendo-ui/timepicker/index

**Comment by converter-user-old** [comment]
I will look at that later but I think we can also add a select (like the normal Input).

**Comment by jerebtw** [comment]
I will wait for https://github.com/mantinedev/mantine/pull/607 to be merged and than work on this 👍🏻

**Comment by rtivital** [comment]
@jerebtw please don't, we will not add this feature, instead we will have a new TimePicker component that allows to pick time similar to Select

**Comment by jerebtw** [comment]
I saw that but is that not a Date and Time picker?

**Comment by rtivital** [comment]
I do not get the question, TimeInput – user **inputs** time, TimePicker – user **picks** time from the given list

**Comment by jerebtw** [comment]
My bad I thought of this https://github.com/mantinedev/mantine/issues/239 but that's a other thing
